src/olympia/files/utils.py

```diff
@@ -513,7 +513,7 @@ def archive_member_validator(archive, member):
 
 
 def _validate_archive_member_name_and_size(filename, filesize):
-    if filename is None or filesize is None:
+    if filename is None or filesize is None or '\\' in filename:
```

Comment:
I would add that check a few lines below instead (line 531, where `filename` is being checked). That gives a better, more accurate error message.

Reply:
Heh. I didn't read the function correctly and I thought it would end right below... Yeah, that makes sense.